app-admin/prelude-manager/prelude-manager-3.1.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils systemd
```

Comment:
If we are deleting la files using `find`, we don't need to inherit `eutils` anymore.